Add Featured Artist Track Updates: Summer 2023 news post (#9746)
ppy/osu-wiki: news/2023/2023-07-22-featured-artist-track-updates-summer-2023.md

---
layout: post
title: "Featured Artist Track Updates: Summer 2023"

> date: 2023-07-22 11:00:00 +0000

Reviewer: ```suggestion
date: 2023-07-22 14:00:00 +0000
```

    <source src="https://assets.ppy.sh/artists/38/Songs/BilliumMoto%20%26%20Dictate%20-%201KARUGA.mp3" type="audio/mpeg">
</audio>

## [ELFENSJoN](https://osu.ppy.sh/beatmaps/artists/69)

[![](https://assets.ppy.sh/artists/69/header.jpg)](https://osu.ppy.sh/beatmaps/artists/69)

These **3** new songs by **ELFENSJoN** have the all the essentials of osu! music: high-paced gothic J-rock and the vocal mastery of veteran utaite *Hanatan*.

If that's not convincing already, you may need to reconsider your preferences.

### ELFENSJoN - DAWN

<audio controls>
    <source src="https://assets.ppy.sh/artists/69/DAWN/ELFENSJoN%20-%20DAWN.mp3" type="audio/mpeg">
</audio>

### ELFENSJoN - ECLIPSE

<audio controls>
    <source src="https://assets.ppy.sh/artists/69/ECLIPSE/ELFENSJoN%20-%20ECLPISE.mp3" type="audio/mpeg">
</audio>

### ELFENSJoN - Magatsu Yami ni Utau

<audio controls>
    <source src="https://assets.ppy.sh/artists/69/Magatsu%20Yami%20ni%20Utau%2FELFENSJoN%20-%20Magatsu%20Yami%20ni%20Utau.mp3" type="audio/mpeg">
</audio>

## [Good Kid](https://osu.ppy.sh/beatmaps/artists/240)

[![](https://assets.ppy.sh/artists/240/header.jpg)](https://osu.ppy.sh/beatmaps/artists/240)

> **Good Kid**'s third EP of hype-filled upbeat rock was released earlier this year (aptly titiled *Good Kid 3*). Since all of their already-licensed songs are already mapped, it only made sense for us to snag **4** of their new tunes.

Reviewer: ```suggestion
**Good Kid**'s third EP of hype-filled upbeat rock was released earlier this year (aptly titled *Good Kid 3*). Since all of their already licensed songs are already mapped, it only made sense for us to snag **4** of their new tunes.
```

</audio>

## [KOAN Sound](https://osu.ppy.sh/beatmaps/artists/49)

[![](https://assets.ppy.sh/artists/49/header.jpg)](https://osu.ppy.sh/beatmaps/artists/49)

Another one of our founding Featured Artists, **KOAN Sound**, is back with **2** extra tracks to round out their catalogue. Both have a special place on osu!: [Strike](https://osu.ppy.sh/beatmapsets/440827) was ranked by [Shiirn](https://osu.ppy.sh/users/465126) back in 2016, while [Forgotten Myths](https://osu.ppy.sh/beatmapsets/643984) has a classic map by [Mir](https://osu.ppy.sh/users/8688812). And both are worth your time.

### KOAN Sound - Strike

<audio controls>
    <source src="https://assets.ppy.sh/artists/49/Forgotten%20Myths%2FKOAN%20Sound%20-%20Strike.mp3" type="audio/mpeg">
</audio>

### KOAN Sound - Forgotten Myths

<audio controls>
    <source src="https://assets.ppy.sh/artists/49/Forgotten%20Myths%2FKOAN%20Sound%20-%20Forgotten%20Myths.mp3" type="audio/mpeg">
</audio>

## [my sound life](https://osu.ppy.sh/beatmaps/artists/252)

[![](https://assets.ppy.sh/artists/252/header.jpg)](https://osu.ppy.sh/beatmaps/artists/252)

> **my sound life** is a side-project of [**solfa**](https://osu.ppy.sh/beatmaps/artists/207), a legend in the realm of visual novel tracks. We've picked up **2** more songs for their listing, including the classic [re:construction](https://osu.ppy.sh/beatmapsets/119431) mapped by [hoLysoup](https://osu.ppy.sh/users/160558) over 10 years ago!

Reviewer: ```suggestion
**my sound life** is a side project of [**solfa**](https://osu.ppy.sh/beatmaps/artists/207), a legend in the realm of visual novel tracks. We've picked up **2** more songs for their listing, including the classic [re:construction](https://osu.ppy.sh/beatmapsets/119431) mapped by [hoLysoup](https://osu.ppy.sh/users/160558) over 10 years ago!
```

- **seatrus - Jikan Sensou**
- **kuro+seatrus - Bellum**
- **seatrus - Brigade**
- **seatrus - DaRk TRiggeR**
- **seatrus - Desintegrate of Eden**
- **seatrus - HydroMe+eor**
- **seatrus - Night Thea+er Etude**
- **seatrus - Scorcher on The Battlefield**
- **seatrus - Shiny Summer Skies**

### seatrus - DOOMSDAY THUNDERTEMPEST

<audio controls>
    <source src="https://assets.ppy.sh/artists/185/Loudly%20Cloudy%2Fseatrus%20-%20DOOMSDAY%20THUNDERTEMPEST.mp3" type="audio/mpeg">
</audio>

### seatrus - Jikan Sensou

<audio controls>
    <source src="https://assets.ppy.sh/artists/185/Mystica%E2%80%8B%E2%80%8Bll%20Refrain/seatrus%20-%20Jikan%20Sensou.mp3" type="audio/mpeg">
</audio>

### seatrus - a MUSIC is worth 1000 words

<audio controls>
    <source src="https://assets.ppy.sh/artists/185/Honey%20Trap%2Fseatrus%20-%20a%20MUSIC%20is%20worth%201000%20words.mp3" type="audio/mpeg">
</audio>

If you thought this was a lot of songs for an individual artist's catalogue update, you're not ready for what's coming.

## [seleP](https://osu.ppy.sh/beatmaps/artists/138)

[![](https://assets.ppy.sh/artists/138/header.jpg)](https://osu.ppy.sh/beatmaps/artists/138)

> Following the release of their latest album *BloodLine*, the composer behind [the hardest map in osu!](https://osu.ppy.sh/beatmapsets/41686) is back with **3** more VOCALOID masterpieces! Two come from that new album, and one more is from their backlog: [Snow Marriage](https://osu.ppy.sh/beatmapsets/1823014) ranked last year by [P_O](https://osu.ppy.sh/users/831979)!

Reviewer: ```suggestion
Following the release of their latest album *BloodLine*, the composer behind "[the hardest map in osu!](https://osu.ppy.sh/beatmapsets/41686)" is back with **3** more VOCALOID masterpieces! Two come from that new album, and one more is from their backlog: [Snow Marriage](https://osu.ppy.sh/beatmapsets/1823014) ranked last year by [P_O](https://osu.ppy.sh/users/831979)!
```

### seleP - Luna: GABBA SIDE

<audio controls>
    <source src="https://assets.ppy.sh/artists/138/BloodLine/seleP - Luna GABBA SIDE.mp3" type="audio/mpeg">
</audio>

### seleP - Noir

<audio controls>
    <source src="https://assets.ppy.sh/artists/138/BloodLine/seleP%20-%20Noir.mp3" type="audio/mpeg">
</audio>

### seleP - Snow Marriage

<audio controls>
    <source src="https://assets.ppy.sh/artists/138/Sanctuary/seleP%20-%20Snow%20Marriage.mp3" type="audio/mpeg">
</audio>

## [solfa](https://osu.ppy.sh/beatmaps/artists/207)

[![](https://assets.ppy.sh/artists/207/header.jpg)](https://osu.ppy.sh/beatmaps/artists/207)

> So uh. This has never happened before. We're adding **over 50** to **solfa**'s library, nearly *tripling* their tracklist, including a bunch of songs with ranked maps that we couldn't pick up before!

Reviewer: ```suggestion
So uh. This has never happened before. We're adding **over 50** to **solfa**'s library, nearly *tripling* their tracklist, including a bunch of songs with Ranked maps that we couldn't pick up before!
```

</audio>

### solfa feat. Shimotsuki Haruka - my sweet heart

<audio controls>
    <source src="https://assets.ppy.sh/artists/207/chronicle%20%7Ecat%20side%7E/solfa%20feat.%20Shimotsuki%20Haruka%20-%20my%20sweet%20heart.mp3" type="audio/mpeg">
</audio>

### solfa feat. Rita - happiness in slavery

<audio controls>
    <source src="https://assets.ppy.sh/artists/207/chronicle%20%7Edog%20side%7E/solfa%20feat.%20Rita%20-%20happiness%20in%20slavery.mp3" type="audio/mpeg">
</audio>

## [Trial & Error](https://osu.ppy.sh/beatmaps/artists/20)

[![](https://assets.ppy.sh/artists/20/header.jpg)](https://osu.ppy.sh/beatmaps/artists/20)

**Trial & Error** makes a return with one more anime-inspired single **Shuuen no Reboot**, available in four different versions! You can figure out what that means for yourself by visiting their Featured Artist listing. 

### Trial & Error - Shuuen no Reboot

<audio controls>
    <source src="https://assets.ppy.sh/artists/20/Songs/Trial%20%26%20Error%20-%20Shuuen%20no%20Reboot.mp3" type="audio/mpeg">
</audio>

## [Wisp X](https://osu.ppy.sh/beatmaps/artists/16)

[![](https://assets.ppy.sh/artists/16/header.jpg)](https://osu.ppy.sh/beatmaps/artists/16)

> Featured Artist veteran **Wisp X** strikes back with **3** more tracks! These chill tunes are a divergence from the usual hyper-intense rhythm game track, like [Coffee](https://osu.ppy.sh/beatmapsets/733211) hosted by [Xinnoh](https://osu.ppy.sh/users/4236057). If that sounds like your style, you're in luck.

Reviewer: ```suggestion
Featured Artist veteran **Wisp X** strikes back with **3** more tracks! These chill tunes are a divergence from the usual hyperintense rhythm game track, like [Coffee](https://osu.ppy.sh/beatmapsets/733211) hosted by [Xinnoh](https://osu.ppy.sh/users/4236057). If that sounds like your style, you're in luck.
```
